docs/docs/02-installation/01-docker.md

```
@@ -19,10 +19,10 @@ mkdir karakeep-app
 
 ### 2. Download the compose file
 
-Download the docker compose file provided [here](https://github.com/absolutepraya/karakeep/blob/main/docker/docker-compose.yml) directly into your new directory.
+Download the docker compose file provided [here](https://github.com/absolutepraya/marka/blob/main/docker/docker-compose.yml) directly into your new directory.
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)